src/server/identity/server/dex_web.go

```diff
@@ -0,0 +1,118 @@
+package server
+
+import (
+	"context"
+	"net/http"
+	"sync"
+
+	"github.com/pachyderm/pachyderm/src/client/identity"
+
+	dex_server "github.com/dexidp/dex/server"
+	logrus "github.com/sirupsen/logrus"
+)
+
+// webDir is the path to find the static assets for the web server.
+// This is always /dex-assets in the docker image, but it can be overriden for testing
+var webDir = "/dex-assets"
+
+// dexWeb wraps a Dex web server and hot reloads it when the
+// issuer is reconfigured.
```

Comment:
Maybe just add a note to the effect of `'issuer' must be the address at which this identity server is reachable from all pachd nodes (unused if 'localhost_issuer' is set in pachd's OIDC IDP config). This value may not be knowable at cluster creation time, or may change if the cluster is reconfigured` (either here or over `issuer`, just so there's something about when/why issuer might be reconfigured)?